src/App.test.js

```diff
@@ -0,0 +1,8 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+import App from './App';
+
+it('renders without crashing', () => {
+  const div = document.createElement('div');
+  ReactDOM.render(<App />, div);
+});
```

Comment:
There's no assertion in this test, so what's the purpose?
 I suggest add a meaningful test